src/mongocrypt-buffer-private.h

```diff
@@ -201,4 +201,13 @@ bool
 _mongocrypt_buffer_steal_from_string (_mongocrypt_buffer_t *buf,
                                       char *str) MONGOCRYPT_WARN_UNUSED_RESULT;
 
+/* _mongocrypt_buffer_copy_from_uint64 initializes @buf from the byte
+ * representation of @value.
+ * Caller must call _mongocrypt_buffer_cleanup.
+ * Endianness of @value is not modified. Use mongocrypt-endian-private.h
+ * to convert endianness beforehand.
+ */
+void
+_mongocrypt_buffer_copy_from_uint64 (_mongocrypt_buffer_t *buf, uint64_t value);
```

Comment:
Not sure what the local convention is, perhaps we always just assume LE since that's what's in the BSON spec, but if we're inventing a new part of the API, it'd be great to be explicit about our endianness since it is not machine dependent.

Reply:
Good point. I have updated this to suffix with `_le` and put the endian conversion in the body of the function.